### DISCUSSION TITLE
Show a window with 3 Pixel size

### DISCUSSION THREAD

**Discussion by FabianGolembiewski-sp** [AvaloniaUI/Avalonia#16966] [Q&A]
Is there a possibility to show a window with a height or width of 3 Pixel? There seem to be any kind of static MinSize of ~30 px.
In WPF this is possible.

What I try to accomplish: I want to show a border as a "marker" rectange around an application that is captured. Therefore I need 4 windows that are used as "borders".

**Comment by timunie** [comment]
Should be possible I think 🤔 

Make sure to setMinWidth to 0 or whatever

**Comment by FabianGolembiewski-sp** [comment]
SystemDecorations="BorderOnly"
ExtendClientAreaChromeHints="NoChrome"
And setting Min/Max Sizes does the trick.

Thanks for your help. 
Have a nice day.

**Reply by timunie** [reply]
It was a bug before so yeah confusing if one was relying on the wrong behavior.

**Comment by FabianGolembiewski-sp** [comment]
One additioal question:
When setting SystemDecorations="BorderOnly" the background is always white.
I think this is because this a color of the Operating System Border, or something like that.

Is there a possibility to change that color?
Or any other ideas?

**Comment by FabianGolembiewski-sp** [comment]
Or another solution for me would be of I could only show client area with custom background color, but with a hight/width of 3 pixel.
